src/main/scala/loamstream/drm/lsf/BacctAccountingClient.scala

```diff
@@ -102,11 +102,12 @@ object BacctAccountingClient {
         node = node,
         queue = queue,
         startTime = startTime,
-        endTime = endTime)
+        endTime = endTime,
+        raw = Some(rawBacctOutput.mkString("\n")))
     }
   }
   
-  private def isHeaderLine(s: String): Boolean = s.startsWith("CPU_T")
+  private def isHeaderLine(s: String): Boolean = s.trim.startsWith("CPU_T")
```

Comment:
Pretty sure I commented on this before, but given that this is an update to all the parsing code, perhaps a CSV parsing library would be better used?

Reply:
The data being parsed here is pretty free-form: it's whitespace-delimited, sort of, but without any quoting around values that contain whitespace.  I don't think it's "officially" CSV/TSV, and in any case, the delimited part is one line in the middle of a bunch of paragraphs.  The output being parsed wasn't intended to be machine-readable. :\